gapis/service/service.proto

```diff
@@ -1481,7 +1481,19 @@ message ValidateDeviceRequest {
 }
 
 message ValidateDeviceResponse {
-  Error error = 1;
+  reserved 1;
+  oneof res {
+    DeviceValidationResult result = 2;
+    Error error = 3;
+  }
+}
+
+message DeviceValidationResult {
+  // Error from validiting the device.
```

Comment:
Make this comment be more explicit that we are talking about a device that didn't pass validation rather than we somehow failed to run validation. Maybe "Message with details of how the device didn't pass validation." or something like it.